.eslintrc.json

```diff
@@ -15,6 +15,7 @@
     "no-alert": "off",
     "import/no-extraneous-dependencies": "off",
     "no-use-before-define": "off",
-    "react/display-name": "off"
+    "react/display-name": "off",
+    "no-shadow": "off"
```

Comment:
Do we want to turn this off for the whole repo? It can catch some pretty annoying bugs. Could we suppress it file by file or line by line instead?